Use __linux__ to detect linux (other unix may not have B2000000)

--- 3rdparty/qbdevice/qbmove_communications.h~	2020-02-14 14:41:31.000000000 +0100
+++ 3rdparty/qbdevice/qbmove_communications.h	2020-03-23 16:35:43.744383338 +0100
@@ -160,7 +160,7 @@
  *  \endcode
 **/
 
-#if !(defined(_WIN32) || defined(_WIN64)) && !(defined(__APPLE__)) //only for linux
+#if defined(__linux__) //only for linux
     void openRS485( comm_settings *comm_settings_t, const char *port_s, int BAUD_RATE = B2000000);
 #elif !(defined(_WIN32) || defined(_WIN64)) && (defined(__APPLE__)) //only for mac
     void openRS485( comm_settings *comm_settings_t, const char *port_s, int BAUD_RATE = 2000000);
